index.html

```diff
@@ -9,43 +9,58 @@
     <link rel="stylesheet" href="style/style.css">
     <title>Portfolio</title>
     <style>
-        section{
-            background: url("header-shapes mobile.png");
+        .info{
+            background: url("style/img/header-shapes\ mobile.png");
             background-size: cover;
+            display: flex;
+            flex-direction: column;
+            align-items: flex-start;
+            padding: 114px 24px;
+            gap: 12px;
+            position: absolute;
+            width: 375px;
+            height: 604px;
+            left: calc(50% - 375px/2);
+            top: 92px;
         }
```

Comment:
- [Optional] Please remove all styling from the HTML file. It is best practice to have all your styles in a `.css` file.

- [Optional] Please in all future projects, do not create a new pull request for a re-review. Always implemented suggested changes and request for a re-review using the same pull request.